### DISCUSSION TITLE
Outlook 2010 and Activesync

### DISCUSSION THREAD

**Discussion by Murmel99** [tine20/tine20#7328] [Q&A]
Hello folks

I’m just testing Tine 2.0 in my local network and I have a problem to get a connection with Outlook 2010 (I know it’s old but all I have currently) and Activsync. Tine is running on a Raspberry Pi 4 with Debian Buster.

http://server/Microsoft-Server-ActiveSync => works
Android Smartphone with Exchange => works
Thunderbird with Plugin => works
Outlook 2019 (evaluation copy) => works
Windows 10 Mail App => works

When creating a profile with Outlook 2010 I get a message like “The connection to the Microsoft Exchange Server is unavailable. Outlook must be online or connected to complete this action.”

The (very old) tool Active Sync Tester tells me “ActiveSync detected, but not correctly configured.”

Is that a general problem with Outlook 2010 or a Tine configuration problem? I would appreciate some hints and tips.

Greetings from Germany,
Peter

**Comment by pschuele** [comment]
hi Peter,
difficult to say what the problem ist with Outlook 2010 ... I think, tine only supports OL2013 and newer versions.
